### PR TITLE
Onboard openshift pipelines on rosa hypershift

### PR DESCRIPTION
**Pull request by shakyav**
[LPTOCPCI-1060](https://issues.redhat.com/browse/LPTOCPCI-1060)
Changes :
1. Added the config for pipelines operator on rosa HyperShift
2. updated the test commands script to not use "oc login" as kubeconfig is already set in the environment, also because oc login with kubeadmin will result in failure on rosa hypershift and rosa sts

### CODE REVIEW

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.15-lp-interop-openshift-pipelines-interop-aws

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          {"step": "rosa-*", "failure_type": "all", "classification":  "Cluster Creation", "jira_project": "LPTOCPCI", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp"]},
+          {"step": "cucushift-hypershift-*", "failure_type": "all", "classification": "Cluster Health Check", "jira_project": "LPTOCPCI", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp"]},
+          {"step": "openshift-pipelines-install-and-tests", "failure_type": "pod_failure", "classification": "Test Failure", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp","interop-tests"]},
+          {"step": "openshift-pipelines-install-and-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "SRVKP", "jira_component": "QA", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp","interop-tests"]}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: "4.14"
+      REGION: us-west-2
```
Comment (Simranpal) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml:48]: I was wondering if REGION is needed?

Comment (shakyav) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml:48]: I think @ascerra can comment better on this, but I think it's to keep all hypershift clusters in one region just for segregation purposes

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      memory: 4Gi
+tests:
+- as: openshift-pipelines-rosa-hypershift
+  cron: 0 6 25 12 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI", "jira_additional_labels": ["4.14-lp","rosa-hypershift-lp","pipelines-lp"]},
```
[the remark this answers is not in the document]
Comment (shakyav) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.11__openshift-pipelines-ocp4.14-lp-rosa-hypershift.yaml]: added

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-rosa-hypershift-openshift-pipelines-rosa-hypershift

**Comment by shakyav** [comment]
closing this , opening a new PR with only latest changes

**Pull request closed without merging**